doc/news/2020-10-30_0.9.3.md

```diff
@@ -222,7 +221,7 @@ The [hashsums are:](https://github.com/ElektraInitiative/ftp/blob/master/release
 - sha1: 02900ef5c8b24cf067930068c62f2ff09b44354d
 - sha256: 5022a6ebf004d892ded03fcf6eb3d223942a7fadd2d68f14d847d1f7f243e1d7
 
-The release tarball is also available signed by Markus Raab using GnuPG from
+The release tarball is also available signed by Mihael Pranjić using GnuPG from
```

Comment:
🙈

I think it makes no sense chaning the template now. Depending on the automated releases we might use a different signature for the next release anyway.

Reply:
> I hope the website is rebuilt quickly.

Maybe we need an extra pipeline (and step during releases) for that? I added a line in #3536.

> I think it makes no sense changing the template now. Depending on the automated releases we might use a different signature for the next release anyway.

Yes, we should directly change the template towards the full automation.